docs/package.json

```diff
@@ -28,14 +28,14 @@
     "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.13.5",
     "@mui/docs": "6.1.10",
-    "@mui/icons-material": "^5.16.9",
-    "@mui/joy": "^5.0.0-beta.48",
-    "@mui/lab": "^5.0.0-alpha.173",
-    "@mui/material": "^5.16.9",
-    "@mui/material-nextjs": "^5.16.8",
-    "@mui/styles": "^5.16.9",
-    "@mui/system": "^5.16.8",
-    "@mui/utils": "^5.16.8",
+    "@mui/icons-material": "https://pkg.csb.dev/mui/material-ui/commit/a0262017/@mui/icons-material",
```

Comment:
These are going to be replaced with a new v5 release.
I'm unsure yet, but we might need to bump the peer dependency `@mui/material` version to the same one in order to avoid issues. 🤔 
However, given that we still support older React versions, maybe it's not mandatory, we could only mention that you need the latest version in the changelog... 🤔